modules/support-collecting-host-network-trace.adoc

```diff
@@ -8,8 +8,15 @@
 
 Sometimes, troubleshooting a network-related issue is simplified by tracing network communication and capturing packets on multiple nodes at the same time.
 
+ifndef::openshift-origin[]
 You can use a combination of the `oc adm must-gather` command and the `registry.redhat.io/openshift4/network-tools-rhel8` container image to gather packet captures from nodes.
 Analyzing packet captures can help you troubleshoot network communication issues.
+endif::openshift-origin[]
+
+ifdef::openshift-origin[]
+You can use a combination of the `oc adm must-gather` command and the `quay.io/openshift/origin-network-tools:latest` container image to gather packet captures from nodes.
+Analyzing packet captures can help you troubleshoot network communication issues.
+endif::openshift-origin[]
```

Comment:
@peer-review-squad Existing text ^^. I updated only the image name in the first sentence.